apps/omg_watcher/test/integration/invalid_exit_test.exs

```diff
@@ -85,6 +86,49 @@ defmodule OMG.Watcher.Integration.InvalidExitTest do
     IntegrationTest.wait_for_byzantine_events([], @timeout)
   end
 
+  @tag fixtures: [:watcher_sandbox, :stable_alice, :child_chain, :token, :stable_alice_deposits]
```

Comment:
why not merge this with the non-deposit case as we discussed? if it is possible and I'm not missing anything here, let's not bloat the integration tests.

Having and challenging an invalid deposit-exit in the previous test scenario seems to be easy to get at 1st glance (just open and challenge another standard exit)